apps/mark-scan/backend/src/app.test.ts

```diff
@@ -45,11 +45,12 @@ import { ElectionState } from './types';
 import {
   mockElectionManagerAuth,
   mockPollWorkerAuth,
+  mockCardlessVoterAuth,
 } from '../test/auth_helpers';
 import { PatConnectionStatusReader } from './pat-input/connection_status_reader';
 import { createWorkspace } from './util/workspace';
 
-const TEST_POLLING_INTERVAL_MS = 5;
+const TEST_POLLING_INTERVAL_MS = 15;
```

Comment:
I'm not really sure why I needed this but it fixed my issue with the mockLoadAndPrint function failing looking for 'presenting_ballot' instead of 'scanning'